python/python/tests/test_async_client.py

```diff
@@ -1242,6 +1242,51 @@ async def test_sdiff(self, redis_client: TRedisClient):
         with pytest.raises(RequestError):
             await redis_client.sdiff([string_key])
 
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_sdiffstore(self, redis_client: TRedisClient):
+        key1 = f"{{testKey}}:1-{get_random_string(10)}"
+        key2 = f"{{testKey}}:2-{get_random_string(10)}"
+        key3 = f"{{testKey}}:3-{get_random_string(10)}"
+        string_key = f"{{testKey}}:4-{get_random_string(10)}"
+        non_existing_key = f"{{testKey}}:5-{get_random_string(10)}"
+
+        assert await redis_client.sadd(key1, ["a", "b", "c"]) == 3
+        assert await redis_client.sadd(key2, ["c", "d", "e"]) == 3
+
+        # Store diff in new key
+        assert await redis_client.sdiffstore(key3, [key1, key2]) == 2
+        assert await redis_client.smembers(key3) == {"a", "b"}
+
+        # Overwrite existing set
+        assert await redis_client.sdiffstore(key3, [key1, key2]) == 2
+        assert await redis_client.smembers(key3) == {"a", "b"}
```

Comment:
Can we have different values to ensure that it does overwrite the existing set? Or is this needed since the following tests overwrites the set as well?

Reply:
Sure, changed this in the latest commit